lexos/processors/prepare/scrubber.py

```diff
@@ -428,6 +294,79 @@ def get_all_punctuation_map() -> Dict[int, type(None)]:
     return punctuation_map
 
 
+def scrub_select_apos(text: str) -> str:
+    """Scrubs all non-word-internal apostrophes from a text.
+
+    :param text: The string to be scrubbed of external apostrophes.
+    :return: The text string, now with only internal apostrophes.
+    """
+
+    # If apos preceded by a non-word character: (?<=[^\w])'
+    # OR apos followed by a non-word character: |'(?=[^\w])
+
+    # Using " " to represent non-word whitespace, "w" to represent a word
+    #     character, and "***" to represent any sequence of any characters,
+    #     this pattern will match:
+    # 1) ***w' *** because the apostrophe is followed by a non-word
+    # 2) *** 'w*** because the apostrophe follows a non-word
+    # 3) *** ' *** because the apos. follows AND is followed by non-words
+    # This will also work for apos. next to other punctuation and multiple
+    #     apos. next to each other, because they count as non-word
+
+    pattern = re.compile(r"(?<=[^\w])'|'(?=[^\w])", re.UNICODE)
+
+    # apply the pattern to replace all external or floating apos with
+    # empty strings
+    scrubbed_text = str(re.sub(pattern, r"", text))
+
+    return scrubbed_text
+
+
+def consolidate_hyphens(text: str) -> str:
+    """Converts all hyphens in a text to the minus sign (-).
+
+    :param text: A string which should have hyphens converted.
+    :return: The text string after all hyphens have been replaced.
+    """
+
+    # All UTF-8 values (hex) for different hyphens and dashes, except the
+    # math minus symbol
+    # All unicode dashes have 'Pd'
+    hyphen_values = ['\u058A', '\u05BE', '\u2010', '\u2011', '\u2012',
```

Comment:
maybe there is a smarter way to do this: http://www.fileformat.info/info/unicode/category/Pd/list.htm